client/app/services/generic-objects-list/generic-objects-list.html

```diff
@@ -0,0 +1,36 @@
+<pf-list-view
+					config="vm.genericObjectsListConfig"
+					items="vm.genericObject.objects"
+					custom-scope="vm">
+					<div class="row generic-objects-container">
+            <div class="col-lg-11 col-md-11 col-sm-11 col-xs-11 generic-objects-title"
+                 ng-click="$ctrl.customScope.toggleExpand(item)">
+              <i class="fa fa-angle-right"
+                 ng-class="{'fa-angle-down': item.isExpanded}"></i>
+                 <img src="{{item.picture.image_href}}" class="generic-object-icon" ng-if="item.picture">
```

Comment:
prefer use of `ng-src` here, while this probably will work most of the time, this should be done for the one time the browser will still try to load the {{path}} string it finds in `<img src="{{path}}">`